.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
 <!--
 Thanks for contributing to _fastlane_! Before you submit your pull request, note that files in the docs folder covering Actions (actions.md and actions/*) and Plugins (available-plugins.md) are auto-generated.
-If this is the case, please modify the documentation at their sources (https://github.com/fastlane/fastlane or plugin repository) and will be updated here regularly.
+If this is the case, please modify the documentation at their sources (https://github.com/fastlane/fastlane or plugin repository) and it will be updated here regularly.
```

Comment:
Or maybe something like:

```suggestion
If this is the case, please modify the documentation at its source (at https://github.com/fastlane/fastlane or plugin repository) and it will propagate here on regular basis.
```

Reply:
I like this!